benchmarks/Elastic.Ingest.Elasticsearch.Benchmarks/Benchmarks/BulkIngestionBenchmarks.cs

```diff
@@ -70,6 +70,6 @@ public void BulkAll()
 		channel.TryWriteMany(_data);
 		channel.TryComplete();
 
-		_waitHandle.WaitOne();
+		//	_waitHandle.WaitOne();
```

Comment:
Did you mean for this to remain uncommented?

Reply:
Reverted changes to the benchmark project, wasn't needed in this PR 👍